core/domain/email_manager.py

```diff
@@ -832,3 +834,9 @@ def send_user_query_email(
         recipient_ids, sender_id, email_intent, email_subject, email_body,
         sender_email, sender_name, bulk_email_model_id)
     return bulk_email_model_id
+
+
+def send_test_email_for_bulk_emails(recipient_id, email_subject, email_body):
+    return _send_email(
+        recipient_id, feconf.SYSTEM_COMMITTER_ID, feconf.BULK_EMAIL_INTENT_TEST,
```

Comment:
sender_id should be the person who actually sends it. Maybe replace recipient_id with tester_id and use it in both sender and recipient fields?

Basically this should look almost exactly like the real email in all respects.